include/constants/pokemon_sprite_visualizer.h

```diff
@@ -1,15 +1,13 @@
-#ifndef GUARD_CONSTANTS_POKEMON_DEBUG_H
-#define GUARD_CONSTANTS_POKEMON_DEBUG_H
+#ifndef GUARD_CONSTANTS_POKEMON_SPRITE_VISUALIZER_H
+#define GUARD_CONSTANTS_POKEMON_SPRITE_VISUALIZER_H
 
 //Defines
-#define DEBUG_MON_BACK_X 62
-#define DEBUG_MON_BACK_Y 80
-#define DEBUG_ICON_X 224
-#define DEBUG_ICON_Y 144
-#define DEBUG_MON_SHINY 0
-#define DEBUG_MON_NORMAL 9
-#define DEBUG_FOLLOWER_X 176
-#define DEBUG_FOLLOWER_Y 128
+#define PSV_MON_BACK_X 62
+#define PSV_MON_BACK_Y 80
+#define PSV_ICON_X 224
+#define PSV_ICON_Y 144
+#define PSV_FOLLOWER_X 176
+#define PSV_FOLLOWER_Y 128
```

Comment:
Based on previous "PSS" confusion, maybe we could change the short prefix from `PSV` to `VISUALIZER`